hackathon_site/dashboard/frontend/src/slices/ui/uiSlice.js

```diff
@@ -15,16 +15,16 @@ const uiSlice = createSlice({
     initialState: initialState,
     reducers: {
         toggleCheckedOutTable: (state) => {
```

Comment:
this is just a general comment but these reducers seem to be called only in one place so why do we have to create a whole slice for these states?

Reply:
The alternative would be to use state local to a component (with the `useState()`) hook - which you're right, we could use.

In this case, we use it so that it's persistent when the component is unmounted because the user changed pages. So if they minimize this table on their dashboard, then go look at the inventory list, then come back, it will stay minimized! With state local just to the table, it would be returned to the default state when you came back to the dashboard page.